documentation/modules/configuring/proc-moving-node-pools.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-moving-node-pools-{context}']
+= (Preview) Moving Kafka nodes between node pools
+
+[role="_abstract"]
+This procedure describes how to move nodes between source and target Kafka node pools without downtime.
+You create a new node on the target node pool and reassign partitions to move data from the old node on the source node pool.
+When the replicas on the new node are in-sync, you can delete the old node.
+
+The steps assume that you are using ZooKeeper for cluster management. 
+The operation is not possible for clusters using KRaft mode.
+
+In this procedure, we start with two node pools:
+
+* `pool-a` with three replicas is the target node pool
+* `pool-b` with four replicas is the source node pool
+
+We scale up `pool-a`, and reassign partitions and scale down `pool-b`, which results in the following:
+
+* `pool-a` with four replicas
+* `pool-b` with three replicas
+
+NOTE: During this process, the ID of the node that holds the partition replicas changes. Consider any dependencies that reference the node ID.
+
+.Prerequisites
+
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
+
+.Procedure
+
+. Create a new node in the target node pool.
++
+For example, node pool `pool-a` has three replicas. We add a node by increasing the number of replicas:
++
+[source,shell]
+----
+kubectl scale kafkanodepool pool-a --replicas=4
+----
+
+. Check the status of the deployment and wait for the pods in the node pool to be created and have a status of `READY`.
++
+[source,shell,subs="+quotes"]
+----
+kubectl get pods -n _<my_cluster_operator_namespace>_
+----
++
+.Output shows four Kafka nodes in the node pool
+[source,shell,subs="+quotes"]
+----
+NAME                        READY   STATUS    RESTARTS
+my-cluster-entity-operator  3/3     Running   0
```

Comment:
Should this line be removed? This seems a bit strange as it is right now => it shows some of the pods which will be there but are unrelated (entity operator) and does not show the pods whihc will be there and are relevant (the `pool-b` pods).

I think to make it less confusing, you should consider:
* Listing all pods (EO, Zoo, Pool-a, pool-b) 
* List the pods relevant to the procedure (pool-a and pool-b)
* List only the pool-a pods about which you talk in this particular point

But the current state is a weird mixture between them.